qt/draw_widget.cpp

```diff
@@ -501,9 +501,9 @@ void DrawWidget::SubmitFakeLocationPoint(m2::PointD const & pt)
       if (loc.m_speedLimitMps > 0)
         speed = "SpeedLimit: " + measurement_utils::FormatSpeedNumeric(loc.m_speedLimitMps, measurement_utils::Units::Metric);
 
-      LOG(LDEBUG, ("Distance:", loc.m_distToTarget + loc.m_targetUnitsSuffix, "Time:", loc.m_time, speed,
+      LOG(LDEBUG, ("Distance:", loc.m_distToTarget.ToString(), "Time:", loc.m_time, speed,
```

Comment:
nit: Can add friend function into Distance:
```
friend std::string DebugPrint(Distance const & d) { return d.ToString(); }
```
And can write here LOG(LDEBUG, (loc.m_distToTarget));